src/kaltura-player.js

```diff
@@ -465,11 +496,23 @@ class KalturaPlayer extends FakeEventTarget {
   }
 
   get ads(): ?AdsController {
-    return this._localPlayer.ads;
+    return this._adsController;
   }
 
   get plugins(): {[name: string]: BasePlugin} {
-    return this._localPlayer.plugins;
+    return this._pluginManager.getAll();
+  }
+
+  get uiComponents(): Array<PKUIComponent> {
```

Comment:
does it still suppose to be an API? doesn't it suppose to be internal for UI?

Reply:
done